### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by MyroslavM**
merge branch to master

### CODE REVIEW

File: app/Http/Controllers/PhotoController.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?php
```
Comment (VladPavliuk) [inline app/Http/Controllers/PhotoController.php]: remove this file

Comment (VladPavliuk) [reply app/Http/Controllers/PhotoController.php]: cause you do not use it

File: database/factories/UserFactory.php [hunk 2]
```diff
@@ -19,8 +19,12 @@
 $factory->define(User::class, function (Faker $faker) {
     return [
         'name' => $faker->name,
+        'last_name' => $faker->name,
+        'age' => $faker->numberBetween(1, 30),
+        'phone_number' => $faker->numberBetween(10, 30),
+        'photo' => Str::random(30),
         'email' => $faker->unique()->safeEmail,
-        'email_verified_at' => now(),
+//        'email_verified_at' => now(),
```
Comment (VladPavliuk) [inline database/factories/UserFactory.php]: remove this commented line

File: resources/views/user/edit-profile.blade.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
+@extends('layouts.master')
+@section('title')
+    edit profile
+@endsection
+@section('content')
+<div class="row">
+    <div class="col-md-4 col-md-offset-4">
+        <h1> Edit Profile</h1>
+    </div>
+{{--</div>--}}
```
Comment (VladPavliuk) [inline resources/views/user/edit-profile.blade.php]: remove these commented lines

File: resources/views/user/edit-profile.blade.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+{{--    <div class="row">--}}
+        <div class="col-md-4 col-md-offset-4">
+            <form action="{{route('user.update-profile')}}" method="post" id="checkout-form" >
```
Comment (VladPavliuk) [inline resources/views/user/edit-profile.blade.php]: why do you need id for this tag?

File: resources/views/user/edit-profile.blade.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                <div class="row">
+                    <div class="col-xs-12">
+                        <div class="form-group">
+                            <label for="name">Name</label>
+                            <input value="{{$user->name}}" type="text" id="name" class="form-control" required name="name">
```
Comment (VladPavliuk) [inline resources/views/user/edit-profile.blade.php]: why do you need id for this tag?

File: resources/views/user/profile.blade.php [hunk 6]
```diff
@@ -5,16 +5,18 @@
             <h1> Your Profile</h1>
         </div>
 
-        <div class="col-md-12 col-md-offset-12">
-            @foreach($users as $user)
-            <img src="{{$user->photo}}" alt="">
+        <div class="col-md-9 col-md-offset-9">
+            <img src="{{$user->photo}}" alt="" height="150px">
```
Comment (VladPavliuk) [inline resources/views/user/profile.blade.php]: fill "alt" attribute's value

File: routes/web.php [hunk 7]
```diff
@@ -50,7 +50,8 @@
             'uses' => 'UserController@postSignin',
             'as'   => 'user.signin'
         ]);
-    });
+
```
Comment (VladPavliuk) [inline routes/web.php]: remove this extra blank line

**Review by MyroslavM — COMMENTED** [review]
i correct all comment

**Pull request merged**